ocaml/xapi/pvs_proxy_control.ml

```diff
@@ -100,7 +100,13 @@ let update_site_on_localhost ~__context ~site ~vdi ?(starting_proxies=[]) ?(stop
 	let proxies = starting_proxies @ (List.set_difference local_running_proxies stopping_proxies) |> List.setify in
 	let proxy_config = metadata_of_site ~__context ~site ~vdi ~proxies in
 	if proxy_config.clients <> [] then
-		Network.Net.PVS_proxy.configure_site dbg proxy_config
+		try Network.Net.PVS_proxy.configure_site dbg proxy_config
+		with _ ->
```

Comment:
I think we should either match a few specific exceptions or log the one that we actually found. It could be not the one we expected.